bigbluebutton-html5/imports/api/2.0/audio/client/manager/index.js

```diff
@@ -0,0 +1,192 @@
+import BaseAudioBridge from '../bridge/base';
+import VertoBridge from '../bridge/verto';
+import SIPBridge from '../bridge/sip';
+
+class CallStates {
+  static get init() {
+    return 'initialized state';
+  }
+  static get echo() {
+    return 'do echo test state';
+  }
+  static get callIntoEcho() {
+    return 'calling into echo test state';
+  }
+  static get inEchoTest() {
+    return 'in echo test state';
+  }
+  static get joinVoiceConference() {
+    return 'join voice conference state';
+  }
+  static get callIntoConference() {
+    return 'calling into conference state';
+  }
+  static get inConference() {
+    return 'in conference state';
+  }
+  static get transferToConference() {
+    return 'joining from echo into conference state';
+  }
+  static get echoTestFailed() {
+    return 'echo test failed state';
+  }
+  static get callToListenOnly() {
+    return 'call to listen only state';
+  }
+  static get connectToListenOnly() {
+    return 'connecting to listen only state';
+  }
+  static get inListenOnly() {
+    return 'in listen only state';
+  }
+  static get reconnecting() {
+    return 'reconecting';
+  }
+}
+
+const ErrorCodes = {
+  CODE_1001: '1001',
```

Comment:
They're not defined in multiple places.

They're defined only in 1 place and imported every else.

In this pull request they are technically defined twice. But thats because there's a 1.1 and 2.0 version of the file